docs/terminal/Sydney TCU/operations.md

```diff
@@ -39,6 +39,9 @@ When assigning levels on descent to aircraft, remember that you will not receive
 
 !!! example
     For an aircraft inbound from the north on the BOREE3A arrival Runway 34R, assign no lower than `A060` until south of the field, then lower as required
+
+!!! tip
+    With aircraft side by side on the RIVET and ODALE STARs, lateral separation will be lost at around 13 DME due to the converging nature of the procedures.  Consider assigning vertically-separated levels to aircraft on each STAR if they are likely to be in close lateral proximity.
```

Comment:
Can we add the stuff here that @LeGiraffe said, and also what J has mentioned previously?

<img width="364" alt="Screen Shot 2022-11-07 at 9 00 31 am" src="https://user-images.githubusercontent.com/7121192/200197509-c4541e7b-c4bc-4d32-997f-5b2b8074681e.png">

<img width="368" alt="Screen Shot 2022-11-07 at 9 00 07 am" src="https://user-images.githubusercontent.com/7121192/200197485-76e5c6a5-2c25-4edf-8f96-3b560cf00efb.png">

Reply:
Yep sounds good

Reply:
if we're going down the route of including this in our TCU procedures, we'll probably need to make similar changes to the ML TCU page as well (all inbounds not via a STAR are given R090X20ML), and im sure other TCUs have similar situations

Reply:
Does ML have a similar arrangement to SY? Coord-less tracking above a certain level?

Reply:
You could include things like "do not assign below A060" until the aircraft has passed abeam the departing runway extended centreline... (27 arbey arrival, not below 6000 until the turn the corner) and (34 arbey arrival, not below A060 until south of 27 extended centreline).

Reply:
nah the R090X20ML requirement is reference the strategic departure/arrivals separation, not coord requirements between DEP and APP